backend/codesirius/problems/models/problem.py

```
@@ -0,0 +1,27 @@
+        title: A CharField representing the title of the problem.
+        description: A TextField representing the description of the problem.
+        tags: A ManyToManyField representing the tags of the problem.
+        languages: A ManyToManyField representing the languages of the problem.
+    """
+
+    title = models.CharField(max_length=200)
+    description = models.TextField(max_length=5000)
+    tags = models.ManyToManyField(Tag, related_name="problems")
+    languages = models.ManyToManyField(Language, related_name="problems")
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)